CONTRIBUTING.md

```diff
@@ -44,3 +44,16 @@ install the following:
 
 Then from the command line navigate to the `/docs` directory and call `make
 html`. The HTML pages will be in `_build/html`.
+
+##### Making a release (needs admin rights)
+1. go to your python environment for `pybv`
+1. run `pip install -U setuptools wheel twine`
+1. update the `__version__` variable in `__init__.py`
+1. commit the change and then git tag the commit with the version
```

Comment:
I believe that if you create a release via the github UI, it'll also tag it, I've found that to be more intuitive for folks

Reply:
That's a good suggestion, thanks!